.dagger/checks.go

```diff
@@ -7,97 +7,115 @@ import (
 )
 
 // Verify that generated code is up to date
-func (dev *DaggerDev) CheckGenerated(ctx context.Context) error {
+func (dev *DaggerDev) CheckGenerated(ctx context.Context) (CheckStatus, error) {
 	_, err := dev.Generate(ctx, true)
-	return err
+	return CheckCompleted, err
 }
 
-func (dev *DaggerDev) CheckReleaseDryRun(ctx context.Context) error {
-	return parallel.New().
-		WithJob("Helm chart", dag.Helm().CheckReleaseDryRun).
-		WithJob("CLI", dag.DaggerCli().CheckReleaseDryRun).
-		WithJob("Engine", dag.DaggerEngine().CheckReleaseDryRun).
-		WithJob("SDKs", func(context.Context) error {
-			type dryRunner interface {
-				Name() string
-				CheckReleaseDryRun(context.Context) error
-			}
+func (dev *DaggerDev) ReleaseDryRun(ctx context.Context) (CheckStatus, error) {
+	return CheckCompleted, parallel.New().
+		WithJob("Helm chart", func(ctx context.Context) error {
+			_, err := dag.Helm().ReleaseDryRun(ctx)
+			return err
+		}).
+		WithJob("CLI", func(ctx context.Context) error {
+			_, err := dag.DaggerCli().ReleaseDryRun(ctx)
+			return err
+		}).
+		WithJob("Engine", func(ctx context.Context) error {
+			_, err := dag.DaggerEngine().ReleaseDryRun(ctx)
+			return err
+		}).
+		WithJob("SDKs", func(ctx context.Context) error {
 			jobs := parallel.New()
-			for _, sdk := range allSDKs[dryRunner](dev) {
-				jobs = jobs.WithJob(sdk.Name(), sdk.CheckReleaseDryRun)
+			for name, fn := range map[string]func(context.Context) (CheckStatus, error){
```

Comment:
We should use a slice here, map iteration is random :cry: 

Also, the `allSDKs` helper isn't used here? I liked that!

Reply:
Mmm, actually in this case I can use `allSDKs` since all checks are from the monolith module. Let me bring that back.